.github/actions/security-scan/action.yaml

```diff
@@ -24,4 +24,5 @@ runs:
         exit-code: '1'
       env:
         # TEMP FIX FOR: https://github.com/aquasecurity/trivy-action/issues/389
-        ACTIONS_RUNTIME_TOKEN: ${{ inputs.github-token }}
+        ACTIONS_RUNTIME_TOKEN: ${{ inputs.GITHUB_TOKEN }}
```

Comment:
Shouldn't we leave this matching the input string as github-token?

Reply:
yeah sorry im just changing it, just doing something different. We need a mirror, ECR is gonna hit exactly the same thing so scratch this, I'll make a mirror for us.

Reply:
Closed pr.